uwblueprint/community-fridge-kw
# feat: admin edit checkins page

## Brief description. What is this change? 

### [Admin Edit Checkin Page](https://www.notion.so/uwblueprintexecs/Admin-Edit-Checkin-Page-5ee6d569e8164760a0fa8bf187c0dc4a)



## Implementation description. How did you make this change?
* Created edit page at `admin/checkin/:id`
* Allows admin to modify time and notes of checkin (in url param)
* Error validation for time range

<img width="696" alt="image" src="https://user-images.githubusercontent.com/69697180/162635784-1dacdf84-b8d8-45a6-aa33-e8d61e78c04f.png">


## Steps to test
1. Create a checkin (http://localhost:3000/admin/create-checkin)
2. Update checkin (http://localhost:3000/admin/checkin/:id)
   * can also test in calendar UI (click "edit" on card)
3. Check that values are updated only after save
4. Check figma designs


## Checklist
